eli5/formatters/text.py

```diff
@@ -159,7 +159,7 @@ def _transition_features_lines(explanation):
     return [
         "",
         "Transition features:",
-        tabulate(tf.coef, headers=tf.class_names, showindex=tf.class_names,
+        tabulate(tf.coef, headers=tf.class_names, showindex=tf.class_names,  # type: ignore
```

Comment:
Looks like mypy is confused by local ``tabulate`` import shadowing the global import, probably a better fix would be to move ``from tabulate import tabulate`` to the top level and import ``tabulate`` from utils as ``eli5_tabulate`` or something like this?

Reply:
Yes, good point. Changed.